cli/collect_metrics.php

```diff
@@ -0,0 +1,63 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace tool_cloudmetrics;
+
+use tool_cloudmetrics\task\collect_metrics_task;
+
+/*
+ * A script to be able to execute the task runner independantly of the task system.
```

Comment:
I'm not opposed to this script existing but is there any main benefit to it when we can run this like:

php admin/cli/scheduled_task.php --execute='\tool_cloudmetrics\task\collect_metrics_task'

Reply:
Removed.